mmf/models/movie_mcan.py

```diff
@@ -68,12 +68,10 @@ def _init_feature_encoders(self, attr: str):
         feature_dim = self.config[attr + "_feature_dim"]
         setattr(self, attr + "_feature_dim", feature_dim)
 
-        encoder_type = feat_encoder.type
-        encoder_kwargs = copy.deepcopy(feat_encoder.params)
-        encoder_kwargs.model_data_dir = self.config.model_data_dir
-        encoder_kwargs.cond_features = self.text_embeddings_out_dim
```

Comment:
Do we intend to remove the `cond_features`?
If we remove it here, can we also remove the cond_features in the movie_mscan's config yaml file?